### ISSUE TITLE
Improve collapse all icon

### ISSUE THREAD

**Issue by vogella** [eclipse-platform/eclipse.platform.images#114]
The most "disruptive" icon in the toolbar (in the dark theme) is currently the collapse all icon. Compare is with the min-max icons which look IMHO really good in the screenshot.

![image](https://github.com/user-attachments/assets/5a79bca3-af12-4968-a866-ba5bf1b8ff28)

@Michael5601 @BeckerWdf  if you agree, could you improve this icon? This would be a very fast quick win,

**Comment by vogella** [comment]
https://github.com/eclipse-platform/eclipse.platform.images/issues/108

**Comment by BeckerWdf** [comment]
So what you say is that it's to bright?
So then also the filter icon is quite bright.

**Comment by HeikoKlare** [comment]
Note that the min/max icons are no classic icons at all. These icons are programatically rendered via an ImageDataProvider.

**Comment by vogella** [comment]
It is too bright and has a blue gradient. I think removing the gradient and making its background a bit transparent would hugely improve the icon.

**Comment by vogella** [comment]
Or

![image](https://github.com/user-attachments/assets/c5961e55-6847-4294-8585-b8c20e2c4fe6)

**Comment by BeckerWdf** [comment]
I will try to improve the icon.

**Comment by BeckerWdf** [comment]
"Package Explorer" has the current Icon "Project Explorer" has my changed icon:

<img width="398" alt="Screenshot 2024-11-29 at 09 46 34" src="https://github.com/user-attachments/assets/0e15ff65-b8f1-4fb4-8674-6552cae61711">

<img width="399" alt="image" src="https://github.com/user-attachments/assets/12e8e8ac-299a-42a1-9db2-365c547517f3">

I have the impression the stroke color is to pale. The hard thing here is that this should look good on light and dark theme.
If I make the stroke color darker then the contrast ist good on the light background but worse on dark background.

**Comment by vogella** [comment]
New icon looks already way better than the old one.

**Comment by BeckerWdf** [comment]
> New icon looks already way better than the old one.

I don't agree. The missing color makes it look like it's disabled - especially in the dark theme.

**Comment by BeckerWdf** [comment]
> Or
> 
> ![image](https://private-user-images.githubusercontent.com/139910/390923227-c5961e55-6847-4294-8585-b8c20e2c4fe6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NjU1MTgsIm5iZiI6MTczMjg2NTIxOCwicGF0aCI6Ii8xMzk5MTAvMzkwOTIzMjI3LWM1OTYxZTU1LTY4NDctNDI5NC04NTg1LWI4YzIwZTJjNGZlNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQwNzI2NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kZTliZjFiYjE2OGI4Yjc5MjI2NTUyOTM0MjQyNzQ1OTI5NmIxNjU2MzRlNTE5MGI4OGE3OTA0NDc0N2M3N2NkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.alsu25bvL7OUUYQZAxUQx2YPwidMgELgc4iJtrR9QvA)
> 
> or
> 
> ![image](https://private-user-images.githubusercontent.com/139910/390923475-c573cd0c-d691-4417-8df9-83daeb2aafc4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4NjU1MTgsIm5iZiI6MTczMjg2NTIxOCwicGF0aCI6Ii8xMzk5MTAvMzkwOTIzNDc1LWM1NzNjZDBjLWQ2OTEtNDQxNy04ZGY5LTgzZGFlYjJhYWZjNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQwNzI2NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNDQ1NDRmY2U2YzY2YzZhNjM2OTIyOTI4NGFhN2MxZTA3MTcyOTc5MDRmNmJhOTFhMzg4OWM2ZDI4YmQ1MmU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.NNcQsKey5mrCk2oz6sUT29OmAGV6g5fYv1cGJcIn5v8)

Some as above:
It's not possible to judge in Inkscape if this looks good or not. One has to see it on the correct background and inside the workbench to see how it really looks like.

**Comment by merks** [comment]
FYI, for some reason your images don't show up:

![image](https://github.com/user-attachments/assets/c145289a-805e-4708-9407-37528d09a642)

**Comment by BeckerWdf** [comment]
> FYI, for some reason your images don't show up:
> 
> ![image](https://private-user-images.githubusercontent.com/208716/391102926-c145289a-805e-4708-9407-37528d09a642.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI4ODg0NTYsIm5iZiI6MTczMjg4ODE1NiwicGF0aCI6Ii8yMDg3MTYvMzkxMTAyOTI2LWMxNDUyODlhLTgwNWUtNDcwOC05NDA3LTM3NTI4ZDA5YTY0Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOVQxMzQ5MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YzczOTIwZWZjNzRjNDI5Mjc1Njc5MjIzZTI2ZmU5MzM0YWMxMTI1MmNkNGExMTg1ZTg2ODk0NDEzZDFmMDhkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.PEc9_C0SPBcvUsmb5hn1USKGgfTRzEX3Vu0T_vj_yqQ)

This is a quote from Lars' images. Do they show up?

**Comment by merks** [comment]
I see it's an issue with quoted images!

![image](https://github.com/user-attachments/assets/13abb33c-2660-45e7-9e3c-6a61326bef62)

My image shows up in Firefox but not in a browser where I've not logged in to GitHub...

**Comment by BeckerWdf** [comment]
Yet another idea. Now with some blue color (no gradient) so that it does not look disabled in the dark theme:

<img width="398" alt="light_blue_lilght" src="https://github.com/user-attachments/assets/34aa425e-71d2-4e08-a56e-467fdaedd078">
<img width="394" alt="light_blue_dark" src="https://github.com/user-attachments/assets/b0433696-447b-4afd-b60d-952f675989d0">

**Comment by vogella** [comment]
I dislike the stronger usage of blue in the new proposal make the icon look wither more like it's for Windows95

**Comment by merks** [comment]
This is just an observation.  I'll bet a lot of downstream projects have copies of the expand-all / collapse-all icons so we're bound do have variations of these in the future within the same IDE, e.g.,

- https://github.com/eclipse-emf/org.eclipse.emf/blob/master/plugins/org.eclipse.emf.codegen.ui/icons/full/ctool16/ExpandAll.png
- https://github.com/eclipse-xtext/xtext/blob/main/org.eclipse.xtext.ui/icons/elcl16/expandall.gif

This is not an argument against change, just an observation ~~(that this is what happens when icons are not API and cannot be reused)~~.

**Comment by BeckerWdf** [comment]
I don't find a solution that is betten compared to what we have today.

**Comment by HeikoKlare** [comment]
> I don't find a solution that is betten compared to what we have today.

This sounds like there is no progress to make here and the issue seems to be rather stalled in general. 

@BeckerWdf @vogella Since this repository is about to be archived, can we close this issue or is there a need for having a follow-up somewhere else?

**Comment by BeckerWdf** [comment]
We can close this.
